protocols/ping/src/handler.rs

```diff
@@ -254,11 +251,11 @@ impl ConnectionHandler for Handler {
             match fut.poll_unpin(cx) {
                 Poll::Pending => {}
                 Poll::Ready(Err(e)) => {
-                    log::debug!("Inbound ping error: {:?}", e);
+                    tracing::error!("Inbound ping error: {:?}", e);
```

Comment:
@eserilev Oh, we shouldn't upgrade these to errors. Can you please retain the log-level here (and further down too?)